### ISSUE TITLE
Onyx Boox Poke Pro: Koreader freezes durring opening a epub/book

### ISSUE THREAD

**Issue by szebrowski** [koreader/koreader#6748]
* KOReader version: 2020.09.35 (DEV)
* Device: Onyx Boox Poke Pro / Android 6.0

#### Issue
I'm trying to open random book and Koreader freezes after some minutes there's modal "Opening 1234324.epub" which has black background (as far as I remember it should have white background. Reader stays on the screen and nothing happens. Koreader freezes.

#### Steps to reproduce
I opened one book, then I opened another one. 

##### `crash.log` (if applicable)

**Comment by pazos** [comment]
crash.log is needed to see what happens.

Also, some questions:

Did you open the ebook from KOReader or from a external FM/app?. In that case which app?.

>  2020.09.35 (DEV)

Did you modify the code somehow or the dev means you're just running a nightly?

**Comment by pazos** [comment]
Just in case you're opening the book from Calibre Companion: see https://www.mobileread.com/forums/showthread.php?t=333393

TL;DR: CC is a crap. If you're relying on the app to handle your books you'll need to downgrade the application to any version lower or equal than 5.4.4.1

**Comment by Galunid** [comment]
Other than @pazos's questions: 
* Do you have app optimization enabled (i.e. Whiten background, enhanced DPI, or something like that)?
* If the question above is "yes", then does it happen if you disable those?

Potentially helpful, in case it's something like #6736 
* Does it happen always, or randomly?
* Does it happen on KOReader version 2020.09

**Comment by szebrowski** [comment]
reinstalled Koreader nightly and it works 🤷‍♀️

**Comment by pazos** [comment]
if you find the issue again please reopen this ticket and answer the questions. Thanks

**Comment by szebrowski** [comment]
sure thing
